app/src/main/java/org/mozilla/fenix/perf/Performance.kt

```diff
@@ -37,4 +44,22 @@ object Performance {
             activity.reportFullyDrawn()
         }
     }
+
+    fun processIntentIfPerformanceTest(intent: Intent, context: Context) {
+
+        if (intent.getBooleanExtra(HomeActivity.EXTRA_IS_PERFORMANCE_TEST, false)) {
+            val batteryStatus = context.registerReceiver(null, IntentFilter(Intent.ACTION_BATTERY_CHANGED))
+            val isPhonePlugged = batteryStatus!!.getIntExtra(BatteryManager.EXTRA_PLUGGED, -1)
```

Comment:
super-nit: I think it's more readable if we do the boolean evaluation up here instead of in the `if`:
```kotlin
val isPhonePluggedViaUSB = batteryStatus!!.getIntExtra(BatteryManager.EXTRA_PLUGGED, -1)
    == BatteryManager.BATTERY_PLUGGED_USB
```

(or whatever the recommended styling is). Here and below.